AboutUs/aboutUs.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <form>
+        <label for="username">User name</label>
+        <input id="username" type="text">
+    </form></br>
+    <label for="password">Password</label>
+    <input id="password" type="password"></br></br>
+    <label>Country</label>
+    <select>
+        <option>India</option>
+        <option>Usa</option>
+        <option>Russia</option>
+        <option>China</option>
+        <option>Dubai</option>
+    </select></br></br>
+    <label for="date_id">Date of birth</label>
+    <input type="date" id="date_id"></br></br>
+    <label for="marital_status">Marital Status</label>
+    <input type="checkbox" id="marital_status"></input></br></br>
+    <label id="gender_checkbox">Gender</label>
```

Comment:
I think you need to use `for` attribute instead of `id`

Reply:
done